src/scripts/install_python.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+yum -y install epel-release wget gcc make tar zlib-devel openssl-devel libffi-devel sqlite-devel		
+
+export PYTHON_VERSION=3.8.16
+wget https://www.python.org/ftp/python/${PYTHON_VERSION}/Python-${PYTHON_VERSION}.tar.xz -P /tmp
+tar xvf /tmp/Python-${PYTHON_VERSION}.tar.xz -C /tmp/
+cd /tmp/Python-${PYTHON_VERSION}
+./configure --enable-optimizations --prefix=/opt/python/${PYTHON_VERSION}
+make altinstall
+cd /tmp
+rm -rf ./Python*
```

Comment:
Overall, this looks good. Just a couple of suggestions:

```suggestion
yum -y install epel-release 
yum -y install wget gcc make tar bzip2-devel zlib-devel xz-devel openssl-devel libffi-devel sqlite-devel ncurses-devel	

export PYTHON_VERSION=3.8.16
wget https://www.python.org/ftp/python/${PYTHON_VERSION}/Python-${PYTHON_VERSION}.tar.xz -P /tmp
tar xvf /tmp/Python-${PYTHON_VERSION}.tar.xz -C /tmp
/tmp/Python-${PYTHON_VERSION}/configure --enable-optimizations
make -C /tmp/Python-${PYTHON_VERSION} -j $(nproc) altinstall
rm -rf /tmp/Python*
```

_Summary of changes:_

1. Adding some missing development libraries needed for Python when compiling from source. 
2. IIRC you need to install `epel-release` before the installing the other packages so that yum will refresh the package list.
3. Removed the `cd` commands so this way we don't need to worry about which `$PWD` we are running out of within the _dispatch_ script.
4. Removed the `--prefix` flag from configure since `make altinstall` will install Python 3.8 alongside the system interpreter. Also, if we put Python 3.8 under _/opt/Python*_, then the onus will be on us to ensure the proper libraries, include files, manpages, etc are in the environment as well.
5. Added the `-j` flag so that we use more jobs when compiling Python. Also, helps us be explicit that we are using as many processors as possible to compile Python.

Reply:
I needed to `cd` into the `/tmp/Python-${PYTHON_VERSION}` folder. Otherwise, the `/tmp/Python-${PYTHON_VERSION}/configure` command will generate the output files into the $PWD no matter what.